CHANGELOG.md

```diff
@@ -97,6 +97,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add TokenManager Interface ([#7452](https://github.com/opensearch-project/OpenSearch/pull/7452))
 - Add Remote store as a segment replication source ([#7653](https://github.com/opensearch-project/OpenSearch/pull/7653))
 - Add descending order search optimization through reverse segment read. ([#7967](https://github.com/opensearch-project/OpenSearch/pull/7967))
+- Workflow to add a checklist for POC issues ([#8033](https://github.com/opensearch-project/OpenSearch/pull/8033))
```

Comment:
I don't think we need a changelog entry for this scenario. The change is for developers and not OpenSearch users.